native/src/routes/ImageViewModal.tsx

```diff
@@ -1,39 +1,40 @@
-import * as React from 'react'
 import { View } from 'react-native'
 import { ImageViewer } from 'react-native-image-zoom-viewer'
 import { NavigationPropType, RoutePropType } from '../constants/NavigationTypes'
-import withTheme from '../hocs/withTheme'
-import { ThemeType } from 'build-configs'
 import { ImageViewModalRouteType } from 'api-client'
+import React, { ReactElement, useContext } from 'react'
+import { ThemeContext } from 'styled-components'
+
+export const SUPPORTED_IMAGE_FILE_TYPES = ['.jpg', '.jpeg', '.png']
```

Comment:
if the supported image file types gonna be use in several files (like above navigateToLink), it should be in a shared constants file